### ISSUE TITLE
Sample-app doesn't mount as part of the initiation (Windows)

### ISSUE THREAD

**Issue by Beer2Brew** [abitvolatile/sample-app#15]
- Windows: 10
- Vagrant Version: 2.2.14
- Virtualbox version: 6.1.16 r140961 

### When launching Vagrant up   #7 
**Error:**
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!
umount /mnt
Stdout from the command:
Stderr from the command:
umount: /mnt: not mounted

- Vagrant SSH
- Can't cd to /sample-app
- Can't mount the guest addition as there is not optical 
- Added Optical but still can't pull the guest addition.

**Comment by ashleymichaelwilliams** [comment]
I’d be curious to know if you ran the command to install the Vagrant plugin for Guest Additions RubyGems package on the host Windows 10 OS.


[Provisioning Documentation](https://github.com/abitvolatile/sample-app#provision-vagrant-centos7-virtual-machine)
```
# Installs VirtualBox Guest Extensions
vagrant plugin install vagrant-vbguest
```

[Vagrant Docs](https://www.vagrantup.com/docs/cli/plugin#plugin-install)


[vagrant-vbguest package docs](https://github.com/dotless-de/vagrant-vbguest#vagrant-vbguest)
> vagrant-vbguest is a Vagrant plugin which automatically installs the host's VirtualBox Guest Additions on the guest system.

**Comment by Beer2Brew** [comment]
yes...
```
C:\Users\jpaul\repo\github\sample-app (master -> origin)
λ vagrant plugin list
vagrant-vbguest (0.28.0, global)
```

If this is confirmed to work on Windows 10 I'll look at some of my other factors. 

```
==> default: Machine booted and ready!
[default] No Virtualbox Guest Additions installation found.
```

What version are you running? There have been known issues of versions that cause problems and the recommend going go an older version

**Comment by jpaulcox** [comment]
```
==> default: Machine booted and ready!
[default] No Virtualbox Guest Additions installation found.
```

What version are you running? There have been known issues of versions that cause problems and the recommend going go an older version

**Comment by Beer2Brew** [comment]
I found that 0.24 is the highest version that works. 

```
vagrant plugin install vagrant-vbguest --plugin-version 0.24
```

**Comment by ashleymichaelwilliams** [comment]
I’ll have to dig more into this on my systems. I’ll check my version and compare to the version you mentioned worked on your end. 

Is there an issue raised yet on the vagrant plugin Repo yet for this already do you happen to know?

**Comment by ashleymichaelwilliams** [comment]
Here is my output

```
MacBook-Pro-4:~ ashleywilliams$ vagrant plugin list
==> vagrant: A new version of Vagrant is available: 2.2.14 (installed version: 2.2.9)!
==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html

vagrant-vbguest (0.24.0, global)
  - Version Constraint: > 0
```

Probably need to do some testing on my Mac to see if the newer plugins experience the same behavior you reported.
